restlight-core/src/main/java/esa/restlight/core/resolver/result/AbstractResponseBodyReturnValueResolver.java

```diff
@@ -103,6 +104,9 @@ public int getOrder() {
                             break;
                         }
                     }
+                    if (Objects.nonNull(serializer)) {
+                        break;
+                    }
```

Comment:
just break with a label for the outer loop like
```java
outerloop:
for (int i=0; i < 5; i++) {
    for (int j=0; j < 5; j++) {
        if (i * j > 6) {
            break outerloop;
        }
    }
}
```

Reply:
amazing!!!  I almost forget this usage;  This also resolve my  recent problem that how to simple break multi loop1